src/bundle/Resources/config/services/controllers.yaml

```diff
@@ -43,3 +43,11 @@ services:
         calls:
             - [ setContainer, [ '@service_container' ] ]
         tags: [controller.service_arguments]
+
+    Ibexa\Bundle\User\Controller\UserInvitationController:
+        autowire: true
+        arguments:
+            $mailSender: '@Ibexa\User\Invitation\MailSender'
```

Comment:
You should inject \Ibexa\Contracts\User\Invitation\InvitationSender instead of concrete implementation.